docs/higher-order-stores.md

```diff
@@ -0,0 +1,57 @@
+Higher-order stores
+===================
+
+A higher-order store is a function that turns a store creating function into a new store creating function:
+
+```
+createStore => createStore'
+```
+
+Look familiar? It's just like the signature for [middleware](middleware.md), only we're wrapping `createStore()` instead of `dispatch()`.
+
+Higher-order stores are a pattern for creating composable Redux extensions. They're much the same as higher-order components in React. They can be as simple as `applyMiddleware()`, or as powerful as the Redux Devtools](https://github.com/gaearon/redux-devtools). There's no limit to the kinds of things you can create.
```

Comment:
should be:  \[Redux Devtools](https://github.com/gaearon/redux-devtools)

missing _[_